packages/styles/scss/components/masthead/_masthead.scss

```diff
@@ -352,6 +352,12 @@ $search-transition-timing: 95ms;
   .#{$prefix}--header__action.#{$prefix}--overflow-menu {
     height: $spacing-09;
   }
+
+  html[dir='rtl'] {
+    .#{$prefix}--overflow-menu-options.#{$prefix}--overflow-menu-options--open {
+        right: 0 !important;
```

Comment:
would this work without `!important`?

Reply:
@jeffchew I tried to avoid it but I had to use it here, otherwise its taking default left as the full width of menu in opposite way.

Reply:
Just FYI; Carbon core encourages the users to use a toolchain to replace `left` with `right`, etc.: https://github.com/carbon-design-system/carbon/blob/v10.13.0/packages/components/docs/g11n.md

Given many components in this library inherits Carbon core components, wondering our library would fit in the same category.

In post-IE11 world, we can use `inset-block-start`: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties